extension/encoding/awslogsencodingextension/internal/unmarshaler/elb-access-log/unmarshaler.go

```diff
@@ -358,6 +358,16 @@ func (f *elbAccessLogUnmarshaler) addToNLBAccessLogs(resourceAttr *resourceAttri
 	recordLog.Attributes().PutStr(AttributeTLSListenerResourceID, nlbRecord.Listener)
 	recordLog.Attributes().PutStr(string(conventions.TLSProtocolVersionKey), nlbRecord.TLSProtocolVersion)
 	recordLog.Attributes().PutStr(string(conventions.TLSCipherKey), nlbRecord.TLSCipher)
+	recordLog.Attributes().PutInt(AttributeELBConnectionTime, nlbRecord.ConnectionTime)
+	recordLog.Attributes().PutInt(AttributeELBTLSHandshakeTime, nlbRecord.TLSHandshakeTime)
+	recordLog.Attributes().PutStr(AttributeELBIncomingTLSAlert, nlbRecord.IncomingTLSAlert)
+	recordLog.Attributes().PutStr(AttributeELBTLSNamedGroup, nlbRecord.TLSNamedGroup)
+	recordLog.Attributes().PutStr(AttributeELBChosenCertARN, nlbRecord.ChosenCertARN)
+	recordLog.Attributes().PutStr(AttributeELBChosenCertSerial, nlbRecord.ChosenCertSerial)
+	recordLog.Attributes().PutStr(AttributeALPNFeProtocol, nlbRecord.ALPNFeProtocol)
+	recordLog.Attributes().PutStr(AttributeALPNBeProtocol, nlbRecord.ALPNBeProtocol)
+	recordLog.Attributes().PutStr(AttributeALPNClientPreferenceList, nlbRecord.ALPNClientPreferenceList)
+	recordLog.Attributes().PutStr(AttributeELBTLSConnectionCreationTime, nlbRecord.TLSConnectionCreationTime)
```

Comment:
Please add a condition to ignore all `-` values. If this was present in this encoding already, then this was a mistake and should be fixed.

We have this check for others, like S3 access log:

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/7ff1f1f742579b8f98c63b7394091c78685dd40e/extension/encoding/awslogsencodingextension/internal/unmarshaler/s3-access-log/unmarshaler.go#L142